### ISSUE TITLE
projen new awscdk-app-ts creates a git subproject

### ISSUE THREAD

**Issue by pflorek** [projen/projen#2223]
While running `npx projen new awscdk-app-ts` it's creating a git submodule which I don't want to. How to disable this feature?

```shell
npx projen@latest --version
0.65.13
```

<img width="451" alt="Bildschirmfoto 2022-11-04 um 15 32 00" src="https://user-images.githubusercontent.com/13916107/200000598-a0f2a760-441f-4c75-862f-3c42bab6f139.png">

Ok, found the trick `npx projen@latest new awscdk-app-ts --projenrc-ts=true --no-git`

Can we make the previous behaviour the default again. If we create a subproject without a parent, then it won't init a git submodule?

**Comment by mrgrain** [comment]
Am I right this only happens when you run `projen new` inside an existing git directory?

I'm not aware of any recent changes in this area. It stands to reason this was always the case and its probably just normal git behavior when `git init` is run inside an existing git project.

**Comment by mrgrain** [comment]
I've got no stakes for either way. I'm sure some people will rely on the fact it currently behaves that way.

**Comment by pflorek** [comment]
Yes, running inside an existing git directory. It's an edge case, normally I would have used the projen subproject. The last time I had this use case is two years ago 😄
